docs/source/03_tutorial/05_package_a_project.md

```diff
@@ -40,6 +40,6 @@ An executable, `kedro-spaceflights`, is also placed in the `bin/` subfolder of t
 
 ### Docker and Airflow
 
-We support the [Kedro-Docker](https://github.com/quantumblacklabs/kedro-docker) plugin for packaging and shipping Kedro projects within [Docker](https://www.docker.com/) containers.
+We support the [Kedro-Docker](https://github.com/kedro-org/kedro-docker) plugin for packaging and shipping Kedro projects within [Docker](https://www.docker.com/) containers.
```

Comment:
`kedro-docker` and `kedro-airflow` now live under plugins: https://github.com/kedro-org/kedro-plugins

Reply:
Great point, thank you! Updated all of these now.